### ISSUE TITLE
Remove the C code in imports and if statements in the BEAST

### ISSUE THREAD

**Issue by galaxyumi** [BEAST-Fitting/beast#488]
Karl: remove all the references to the C code, both in imports and if statements. That C code is no longer present in the current version of the repository and so it is not possible to ever use it (without going to previous versions and getting it back). Best to just remove it all - more consistent and clean.

**Comment by karllark** [comment]
Of course, for each bit of C code removed we should confirm that it is not used.  I don't think any is and it should be easy to check this.   But good to be sure.  :-)
